lib/mongoid/clients/factory.rb

```diff
@@ -78,9 +87,8 @@ def driver_version
         Mongo::VERSION.split('.')[0...2].map(&:to_i)
       end
 
-      def options(configuration)
-        config = configuration.dup
-        options = config.delete(:options) || {}
+      def options(opts)
```

Comment:
Did the API of this method change? I think a brief docstring would be helpful to understand what this method is supposed to receive in the opts.

Reply:
Added some docstring.